### ISSUE TITLE
I'm Retiring.

### ISSUE THREAD

**Issue by WillSullivan** [WillSullivan/idownvotedbecause#131]
Due to the recent issues with Stack Overflow, I've decided to retire this project. Or, rather, retire my participation with it.

I'd be happy to pass this repo on to others who want to keep the site alive to help folks. If you'd like to volunteer, please leave a comment on this issue. I'll give folks who have contributed to the site in the past the first crack.

As for the idownvotedbecau.se domain, I'll have to see about how to transfer that. 

There's no rush on this. I'd like to get it done by January next year.

**Comment by praveenscience** [comment]
@WillSullivan Happy to take it over mate! 😊

**Comment by open-gdsn** [comment]
Yes, stackoverflow wants to be more new user friendly but honestly they lack the tools for people trying to help out. I agree (to some extend) that the hostname might be a little different, but I'm still very surprised stackoverflow does not have something like this themselves. Hope it continues in some way

**Comment by EricWu91** [comment]
@WillSullivan I'm willing to keep it up too! ✋

**Comment by eaaliprantis** [comment]
@WillSullivan I'm willing to continue it if no one has

**Comment by DCCoder90** [comment]
@WillSullivan Are you going to keep this going?  If not I'm willing to take it on if you have to pass on the torch.  If not, thanks for the resource!

**Comment by WillSullivan** [comment]
I am going to pass it on, have looked into transferring the domain, it's just low on my list of things that depress me but I'll eventually have to deal with eventually. And I need to figure out how I determine who to cede the domains to.

**Comment by DCCoder90** [comment]
@WillSullivan I understand, hope what ever is going on for you gets better.

**Comment by praveenscience** [comment]
> I am going to pass it on, have looked into transferring the domain, it's just low on my list of things that depress me but I'll eventually have to deal with eventually. And I need to figure out how I determine who to cede the domains to.

@WillSullivan I completely feel you man. I have been having a lot of domains and yes, I know how it feels. Happy to take over and if you wanna make sure it needs to stay alive! 🤘🏻 I know it's a hard decision... Let us all know! 😔 We'll take care of it.

**Comment by thetayloredman** [comment]
@WillSullivan Willing to keep it up! ✋

**Comment by JavascriptLearner815** [comment]
@WillSullivan Hype train! ✋🏻

**Comment by gitcnd** [comment]
Let it drop.  Downvoters are a disease that cause helpful people to cease sharing their knowledge - they need to be eliminated, not supported.

**Comment by megargayu** [comment]
@gitcnd That's an example of a "bad downvoter", and not at all what the point of a downvote is. The point of a downvote is to indicate to other answerers that this question is not well written, needs more information, etc. This is also why when you downvote a post on SO, a popup appears - 
> Please consider adding a comment if you think this post can be improved.

Downvoting is not meant to discourage people and to make them stop asking questions; it's more for constructive criticism and not wasting everyone's time. If someone has put absolutely 0 effort into making a question, or it's so misinformative that you can't understand it, then it's a good idea to put a downvote and add a comment telling the question asker what needs to change. upvoting a comment made by someone else about what the question needs/doesn't need, or flagging the question. However, when a good question appears, then people upvote the question so more people see it and can answer it. 

https://meta.stackoverflow.com/questions/252677/when-is-it-justifiable-to-downvote-a-question

**Comment by 7heMech** [comment]
@WillSullivan have you found someone yet?

**Comment by WillSullivan** [comment]
> @WillSullivan have you found someone yet?

Sadly I haven't. I should have time at the end of the year, and should get off my butt and do it. I have a metric I'll use to decide who from those that raised their hands, I just need to take the time to do it.

**Comment by petdance** [comment]
I think this is a great idea, and I'd like to see it made more general. The problem of people not asking questions is bigger than just StackOverflow. I've had ideas rolling around in my head about making a similar site, from the point of view of "here's how to ask" with details on specific things they should encourage.  https://github.com/petdance/asksmart

I don't think I'd want to maintain the I Downvoted site myself, but I put myself out here as someone who is very interested in this problem of rookies not knowing how to ask questions, and how to help them.

**Comment by ssdrive-flaktra** [comment]
I could also maintain the repository. 🤚

**Comment by gitcnd** [comment]
> @gitcnd That's an example of a "bad downvoter", and not at all what the point of a downvote is. 

Not sure what universe you're living in?  99% of downvotes are toxic attacks against someone, and have nothing whatsoever to do with whatever the "point" was supposed to be.

It 100% does, utterly and completely, discourage everyone.  I'm a 40+ year expert in a pile of ICT topics, yet I now consciously avoid going anywhere near that utterly toxic community of opinioned idiot ingrates.

**Comment by WillSullivan** [comment]
I know I've been ignoring this repo and apparently tons of convos that I haven't gotten notifications about. Still willing to transfer ownership. I'll be contacting some folks in the coming weeks and seeing about getting that done so the pile of PRs can get actually processed. 

[edit: was looking for something and happened upon the code of conduct and saw it was linked to SO's code of conduct] Ugh, why did I link "be nice" from my code of conduct to the now corporate shitshow that is the SO code of conduct. Why you have to vomit an absolute river of garbage just to say "be nice" I will never understand. Trying to justify your salary? Fucking bring in donuts on wednesday. 

Anyhow, hoping folks who care about the idea of this repo who want to help people write good questions that attract good quality answers will ping me again and let's get this done.

**Comment by gitcnd** [comment]
Let it drop. A.I. has rendered the toxic sh!tshow of "online help sites" utterly useless, or at least what was left of them after all the toxic downvote-diseased cancel-culture woke morons polluted it all with their hate. 

"nice" really only works face-to-face, so most online people are not nice, and telling them to be so, does the opposite of what you want - it has no effect on the sh!theads, and turns any nice newcomers *into* sh!theads as they try to police the niceness policy.

If you've read this far - you're on the wrong web site - go to the right place: perplexity.ai

**Comment by smathy** [comment]
@WillSullivan ping

**Comment by WillSullivan** [comment]
> @WillSullivan ping

Noted.

**Comment by tvqt** [comment]
@WillSullivan can we get a progress update on where this transfer process is at? It's been five years since you retired, it's insane this is still going on

**Comment by WillSullivan** [comment]
> @WillSullivan can we get a progress update on where this transfer process is at? It's been five years since you retired, it's insane this is still going on

Still looking for someone serious about it to take over. They will need to take control of the godaddy domains as well (free, but you will need to pay for future renewals). Serious candidates who actively want to take over need to start bugging me... Like you did.

**Comment by NullDev** [comment]
@WillSullivan If you still haven't found someone: I'd be willing to maintain this project. I have contributed in the past and don't mind paying for domain renewals. I also am still active on StackOverflow (even though I probably shouldn't...)
